src/Registry/Registry.jl

```diff
@@ -250,6 +253,7 @@ function download_registries(io::IO, regs::Vector{RegistrySpec}, depot::String=d
             end
         end
     end
+    end
```

Comment:
This is so far from the start so maybe nice to add a comment:
```suggestion
    end # mkpidlock
```